packages/@aws-cdk/aws-events/test/test.rule.ts

```diff
@@ -618,7 +618,7 @@ export = {
 
       test.throws(() => {
         rule.addTarget(new SomeTarget('T', resource));
-      }, /You need to provide a concrete account for the source stack when using cross-account or cross-region events/);
+      }, /You need to provide a concrete region/);
```

Comment:
It feels like for all the code you changed (+added), there needs to be more modification in the tests than this. Specifically, the change of which Role to use feels like it deserves to be codified here.

Reply:
The test that tests useful things was added was in the CodePipeline library. Will add one here as well though.